### PR TITLE
(fix) O3-2579 Lab results: Too much grey space

### PR DESCRIPTION
**Pull request by suubi-joshua**
## Requirements

- [x] This PR has a title that briefly describes the work done including the ticket number. If there is a ticket, make sure your PR title includes a [conventional commit](https://o3-docs.openmrs.org/docs/frontend-modules/contributing.en-US#contributing-guidelines) label. See existing PR titles for inspiration.
- [ ] My work conforms to the [OpenMRS 3.0 Styleguide](https://om.rs/styleguide) and [design documentation](https://om.rs/o3ui).
- [ ] My work includes tests or is validated by existing tests.

## Summary
Removed Shadow from test results table in timeline

## Related Issue

https://openmrs.atlassian.net/browse/O3-2579

## Other
After changes
[Screencast from 07-17-2024 12:37:13 AM.webm](https://github.com/user-attachments/assets/408bee79-d543-4279-918d-4f7e59d87351)

[Screencast from 07-17-2024 12:37:13 AM.webm](https://github.com/user-attachments/assets/95fe2c6d-1ffb-41a4-9462-ff0f7664c160)

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -23,7 +23,7 @@
     "zod": "^3.22.2"
   },
   "devDependencies": {
-    "@openmrs/esm-framework": "next",
+    "@openmrs/esm-framework": "^5.6.1-pre.2069",
```
Comment (sherrif10) [inline package.json:26]: Revert this back to `next`

File: package.json [hunk 2]
```diff
@@ -60,7 +60,7 @@
     "jest-environment-jsdom": "^29.7.0",
     "lint-staged": "^14.0.1",
     "lodash": "^4.17.21",
-    "openmrs": "next",
+    "openmrs": "^5.6.1-pre.2069",
```
Comment (sherrif10) [inline package.json:63]: Even here

File: yarn.lock [hunk 3]
```diff
@@ -18243,7 +18243,7 @@ __metadata:
     yargs: "npm:^17.6.2"
   bin:
     openmrs: ./dist/cli.js
-  checksum: 10/7d4b13b33cdfabbc3e0362682aeb81a71a770133d81bc35ad232e6d0e0bc7e5f665fbc5472020c7bbd661c01a6aa342cd1012a64b0970dc2c7410c8a87069c14
+  checksum: 10/4579c3ec89aff07d47da479c6ec088acff96c735b130e8335e3a0f9244412927ec1eaada9b94d6181841381cbeb8fb5d49552f259ac0b20a76d1b0b3581882c4
```
Comment (sherrif10) [inline yarn.lock:18246]: i would expect this ticket to only involve scss classes only basing on the ticket title.

File: package.json (hunk 1 above)
Comment (suubi-joshua) [reply package.json:26]: okay

File: yarn.lock (hunk 3 above)
Comment (suubi-joshua) [reply yarn.lock:18246]: Yes and I did only edit the scss file but some changes were made when I updated framework and also pulled latest changes from upstream
@sherrif10

**Comment by suubi-joshua** [comment]
I am going to close this PR and open another cleaner one @sherrif10

**Pull request closed without merging**